### ISSUE TITLE
ggtree dropping terminal lengths

### ISSUE THREAD

**Issue by jeremycg** [GuangchuangYu/ggtree#4]
Hi, thanks so much for this package! It's a million times easier than customizing every tree.

I have a weird issue where ggtree is not plotting a single terminal branch - here's the steps to reproduce:

```{r}
library(ape)
library(ggtree)
tr<-read.tree(text="(C. sp. 1:0.356,(C. plicata:0.3,((C. guadeloupensis:0.223,((C. portoensis:0.158,C. virilis:0.201):0.005,((C. sp. 8:0.169,(C. angaria:0.031,C. castelli:0.037):0.131):0.073,(C. drosophilae:0.027,C. sp. 2:0.03):0.161):0.047):0.033):0.074,((C. kamaaina:0.14,(C. japonica:0.16,((C. imperialis:0.082,C. afra:0.182):0.009,((C. yunquensis:0.086,C. nouraguensis:0.085):0.038,C. macrosperma:0.159):0.057):0.012):0.059):0.041,(C. elegans:0.166,(((C. brenneri:0.129,C. doughertyi:0.111):0.042,(C. wallacei:0.083,C. tropicalis:0.07):0.033):0.013,(C. remanei:0.129,(C. sp. 5:0.127,(C. briggsae:0.047,C. nigoni:0.032):0.088):0.02):0.017):0.046):0.02):0.195):0.07):0.259);")
plot(tr)
ggtree(tr)+geom_text(aes(label=label),hjust=-0.05,fontface="italic")+geom_text(aes(label=branch.length, x=branch), size = 3, 
              vjust=-0.5, color="#F06C45") +
      scale_x_continuous(breaks=seq(0, 60, 5))
```

C. imperalis - in the middle is not having its correct length plotted out - the branch length calculator seems to see it fine, but no length is plotted. I've tried deleting and adding branches, and there always seems to be one that is not plotted.

**Comment by GuangchuangYu** [comment]
Thanks for your bug report.

This bug has been fixed in version >=0.99.27

**Comment by jeremycg** [comment]
Thanks a million, confirmed fixed.
